pkg/controller/datavolume-controller.go

```diff
@@ -2803,15 +2804,12 @@ func (r *DatavolumeReconciler) garbageCollect(dataVolume *cdiv1.DataVolume, pvc
 }
 
 func (r *DatavolumeReconciler) detachPvcDeleteDv(pvc *corev1.PersistentVolumeClaim, dv *cdiv1.DataVolume, log logr.Logger) error {
-	if !IsSucceeded(pvc) {
+	if dv.Status.Phase != cdiv1.Succeeded {
```

Comment:
I am fairly certain the warm migration flows are now broken because they depend on this annotation `pvc.Annotations[AnnCurrentCheckpoint] == ""` to determine if they actually succeeded the final check point or not.

Reply:
I'll check, but where are the ftest failures?

Reply:
I believe we have most warm migration tests disabled because they are pretty flaky already https://github.com/kubevirt/containerized-data-importer/blob/main/tests/datavolume_test.go#L781

Reply:
Ok, so I'll re-check and/or fix it in Sunday.

Reply:
@awels no worries, import dv is succeeded only if pvc pod phase is succeeded and no current checkpoint
https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/controller/datavolume-controller.go#L2082-L2097